_includes/docs/user-guide/image-gallery.md

```diff
@@ -14,6 +14,18 @@ Users can easily upload, organize, and select images to customize their interfac
 ![image](/images/user-guide/image-gallery/image-gallery-ce.png)
 {% endif %}
 
+{% unless docsPrefix == 'paas/' %}
+## Configuration
+
+Images are cached in the browser, and by default browser will make a request to ThingsBoard for each loaded image to verify that the cached version is up to date.
+If your images are not updated frequently, you might configure cache control for system or/and tenant images for faster image loading.
+To do so, specify *CACHE_SPECS_IMAGE_SYSTEM_BROWSER_TTL* and *CACHE_SPECS_IMAGE_TENANT_BROWSER_TTL* environment variables.
```

Comment:
The description is not clear. Please work with @ViacheslavKlimov to understand all the aspects and write a much more clear description. Also add examples of the env variable values:

```
For example, setting CACHE_SPECS_IMAGE_SYSTEM_BROWSER_TTL to 12345 will force *system* images to be cached in the user's browser for 12345 (milliseconds or seconds, I don't remember) 
```